httpclient5/src/main/java/org/apache/hc/client5/http/impl/cookie/LaxMaxAgeHandler.java

```diff
@@ -68,8 +68,8 @@ public void parse(final SetCookie cookie, final String value) throws MalformedCo
             } catch (final NumberFormatException e) {
                 return;
             }
-            final Date expiryDate = age >= 0 ? new Date(System.currentTimeMillis() + age * 1000L) :
-                    new Date(Long.MIN_VALUE);
+            final Instant expiryDate = age >= 0 ? Instant.now().plusSeconds(age * 1000L) :
```

Comment:
@arturobernalg This looks wrong. Should not it be `Instant.now().plusSeconds(age)` or `Instant.now().plusMillis(age * 1000L)`?